Tensile/Tests/nightly/vector_width/sgemm_nn_asm.yaml

```diff
@@ -43,3 +43,9 @@ BenchmarkProblems:
       BenchmarkFinalParameters:
         - ProblemSizes:
           - Range: [ [127,1,129], 0, [4], [63,1,65] ]
+          - Exact: [1024, 2, 1, 4096]
```

Comment:
Have you tested that this fails without the other changes?

Reply:
Yes I have.